crates/bevy_ecs/Cargo.toml

```diff
@@ -11,39 +11,85 @@ categories = ["game-engines", "data-structures"]
 rust-version = "1.81.0"
 
 [features]
-default = ["bevy_reflect"]
-trace = []
+default = ["std", "bevy_reflect", "async_executor"]
+std = [
+  "bevy_reflect?/std",
+  "bevy_tasks/std",
+  "bevy_utils/std",
+  "bitflags/std",
+  "concurrent-queue/std",
+  "disqualified/alloc",
+  "fixedbitset/std",
+  "indexmap/std",
+  "serde?/std",
+  "nonmax/std",
+  "arrayvec?/std",
+  "log/std",
+]
+trace = ["std", "dep:tracing", "bevy_utils/tracing"]
 multi_threaded = ["bevy_tasks/multi_threaded", "arrayvec"]
 bevy_debug_stepping = []
-serialize = ["dep:serde"]
+serialize = ["dep:serde", "bevy_utils/serde"]
 track_change_detection = []
 reflect_functions = ["bevy_reflect", "bevy_reflect/functions"]
-detailed_trace = []
+detailed_trace = ["trace"]
```

Comment:
Comments and maybe some grouping would make these features a lot easier to follow. There's a ton being added here, and it's hard to understand why they're needed and what they each do.

Reply:
Agreed. I've reorganised the features into distinct sections with short descriptions. It's not quite the same as how other crates (e.g., `bevy_internal`) document their features, but I believe this is more appropriate and easier to understand. But I am happy to change it based on feedback!